infra/helm/meshdb/values.yaml

```diff
@@ -47,7 +43,13 @@ meshweb:
     tag: main
   podSecurityContext: {}
   securityContext: {}
-  resources: {}
+  resources:
+    limits:
+      cpu: 1
+      memory: 512Mi
+    requests:
+      cpu: 200m
+      memory: 350Mi
```

Comment:
Pin this guy to 512Mi. 200m is _probably_ fine as a request, as I think most of the CPU is on startup with the tracer.